Makefile

```diff
@@ -341,7 +341,7 @@ beautify: virtualenv/bin/fixjsstyle
 spritegen: $(SPRITE_GENERATED_FILES)
 
 $(COMPILED_JSX_FILES): $(JSX_FILES)
-	jsx --no-cache-dir -x jsx . .
+	# jsx --no-cache-dir -x jsx . .
```

Comment:
Can we Chan up as the follow up then please? My understanding was that the
jsx work was in branches not hitting trunk.
On Apr 23, 2015 3:46 PM, "Jeff Pihach" <notifications@github.com> wrote:

> In Makefile
> <https://github.com/juju/juju-gui/pull/719#discussion_r28996493>:
>
> > @@ -341,7 +341,7 @@ beautify: virtualenv/bin/fixjsstyle
> >  spritegen: $(SPRITE_GENERATED_FILES)
> >
> >  $(COMPILED_JSX_FILES): $(JSX_FILES)
> > -	jsx --no-cache-dir -x jsx . .
> > +	# jsx --no-cache-dir -x jsx . .
>
> I felt that just removing the line of code will leave some odd hanging
> code remaining and I didn't want the branch to creep to much to fully
> remove the jsx stuff so I elected to comment it out as a interim step.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/juju/juju-gui/pull/719/files#r28996493>.
>